hls-plugin-api/src/Ide/Plugin/ConfigUtils.hs

```diff
@@ -28,7 +28,7 @@ import           Language.LSP.Types
 -- | Generates a default 'Config', but remains only effective items
 pluginsToDefaultConfig :: IdePlugins a -> A.Value
 pluginsToDefaultConfig IdePlugins {..} =
-  -- Use 'ix' to look at all the "haskell" keys in the outer value (since we're not
+  -- Use 'key' to look at all the "haskell" keys in the outer value (since we're not
   -- setting it if missing), then we use '_Object' and 'at' to get at the "plugin" key
   -- and actually set it.
   A.toJSON defaultConfig & key "haskell" . _Object . at "plugin" ?~ elems
```

Comment:
@michaelpj i changed `ix` for `key` here for readability but maybe you wanted to use `ix` on porpuse

Reply:
nah, it's fine. `ix` is maybe more familiar to people since it's from `lens` but NBD